### DISCUSSION TITLE
Have Project Of The Weeks show as a spotlight somewhere on the website

### DISCUSSION THREAD

**Discussion by intergrav** [modrinth/.github#22] [Feature requests]
### Describe the solution you'd like

POTW should show on the front page/home page or have it's own page. This would bring more light to those projects

Here is a concept I made:

![image](https://user-images.githubusercontent.com/42325132/181403902-21fa1f99-c024-44c3-8645-cf796d539f3f.png)

I think you should also be able to look at old POTWs

This could be implemented in any way, but this one was my idea

![image](https://user-images.githubusercontent.com/42325132/181405567-8801933c-8fae-4617-a1dd-e387893efada.png)

**Comment by youmukonpaku1337** [comment]
looks poggers and sounds poggers, would be cool
